src/App/App.js

```diff
@@ -64,6 +65,11 @@ export class App extends Component{
     })
   }
 
+  async componentDidMount() {
+    const promise = await fetcher.fetchAllMovies();
+    this.setState({movies: promise.movies})
```

Comment:
I think we change promise to be more descriptive in the next branch. Just adding a comment to note that. Also, `this.setState({movies: promise.movies})` changes to `this.setState({ movies })`